translations/monero-core_de.ts

```diff
@@ -1881,12 +1872,12 @@ Please upgrade or connect to another daemon</source>
     <message>
         <location filename="../wizard/WizardMain.qml" line="168"/>
         <source>A wallet with same name already exists. Please change wallet name</source>
-        <translation>Ein Wallet mit diesem Namen ist bereits vorhanden. Bitte ändere den Wallet-Namen</translation>
+        <translation>Ein Wallet mit diesem Namen ist bereits vorhanden. Bitte ändere den Namen des Wallets</translation>
     </message>
     <message>
         <location filename="../wizard/WizardMain.qml" line="176"/>
         <source>Non-ASCII characters are not allowed in wallet path or account name</source>
-        <translation type="unfinished">Nur ASCII Zeichen sind im Walletpfad oder im Walletname erlaubt.</translation>
+        <translation>Nur ASCII Zeichen sind im Walletpfad oder im Walletname erlaubt.</translation>
```

Comment:
The translation has a period at the end, but not the base english.
Seems one or the other should be changed.